app/js/actions/createLabOrder.js

```diff
@@ -0,0 +1,35 @@
+import 'babel-polyfill';
```

Comment:
I need to require this for the use of async and await on this file

Reply:
Hey @Efosaok 
How about you import the `babel-polyfill` in the entry point of this client codebase so that it can be used in other parts of the codebase without re-importing.